src/content/docs/apm/agents/net-agent/other-installation/understanding-net-agent-environment-variables.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Understanding required .NET agent environment variables
+tags:
+  - Agents
+  - NET agent
+  - Other installation
+redirects:
+---
+
+Our .NET agent is a .NET CLR profiler and relies on your app's environment variables to connect your .NET app with New Relic. To learn more about which environment variables the .NET agent uses, see the sections below. 
```

Comment:
How is this revision of the intro statement? @nr-ahemsath

Reply:
Looks good!